src/common/CustomTable/CustomTable.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { EuiBasicTable } from '@elastic/eui';
+
+const CustomTable = ({
+  items, columns, sorting, onChange,
```

Comment:
You need `...rest` here and provide it to `EuiBasicTable`.

Reply:
Fixed